SPECS/grubby/grubby.spec

```diff
@@ -69,15 +67,15 @@ users with existing grubby users.
 %install
 %make_install mandir=%{_mandir} sbindir=%{_sbindir} libexecdir=%{_libexecdir}
 
-mkdir -p %{buildroot}%{_libexecdir}/{grubby,installkernel}/ %{buildroot}%{_sbindir}/
+# We do not supply the installkernel script from grubby. Instead users should
```

Comment:
Should we add a `Requires: installkernel` so that make install still works properly?

Reply:
I like this suggestion. Let me try it out.